var/spack/repos/builtin/packages/py-onnxruntime/package.py

```diff
@@ -52,7 +56,7 @@ class PyOnnxruntime(CMakePackage, PythonExtension):
     # https://github.com/cms-externals/onnxruntime/compare/5bc92df...d594f80
```

Comment:
py-coloredlogs is required now

Reply:
Still missing

Reply:
Yeah, this has been a difficult one to get working. I have 15.1.0 working, but not yet any of the intermediate versions...